its/ruling/src/test/resources/expected/php-S1186.json

```diff
@@ -0,0 +1,604 @@
+{
+'project:PHP_CodeSniffer/src/Files/File.php':[
+287,
+],
+'project:PHP_CodeSniffer/src/Standards/Generic/Sniffs/Arrays/ArrayIndentSniff.php':[
+39,
+],
+'project:PHP_CodeSniffer/src/Standards/Generic/Sniffs/NamingConventions/ConstructorNameSniff.php':[
+123,
+],
+'project:PHP_CodeSniffer/src/Standards/PEAR/Sniffs/NamingConventions/ValidVariableNameSniff.php':[
+77,
```

Comment:
That's a FP which shows that the regular expression we use for comments is not correct.

Reply:
We still raise an issue there. Is it expected?

Reply:
Miss to update the expected test results